projects/agent/watchers/workflow.py

```diff
@@ -125,9 +125,13 @@ def update_status(workflow_manifest, session):
             if status_message is not None:
                 status_message = str(status_message)
 
-        session.query(models.Operator) \
-            .filter_by(uuid=operator_id) \
-            .update({"status": status, "status_message": status_message})
+        # Discarding recurrent messages
+        recurrent_messages = ["ContainerCreating", ]
```

Comment:
@andreluiz27 acho que fica legal colocar essa lista lá no início do arquivo, como uma "constante": `RECURRENT_MESSAGES = ["ContainerCreating", ]`